src/consensus/params.h

```diff
@@ -27,7 +27,7 @@ enum DeploymentPos
 struct BIP9Deployment {
     /** Bit position to select the particular bit in nVersion. */
     int bit;
-    /** Start MedianTime for version bits miner confirmation. Can be a date in the past */
+    /** Start MedianTime for version bits miner confirmation. Can be a date in the past. Negative numbers mean hardcoded height. */
```

Comment:
I would expand this comment a bit.  I think its a bit ugly and counterintuitive that positive numbers are times and negative numbers are heights.  I think I'm ok with the hack, but it should be more clearly identified

Reply:
Better now?